mirage/apt/apt_inputs.py

```diff
@@ -338,6 +340,39 @@ def create_input_table(self, verbose=False):
         ascii.write(Table(self.exposure_tab), self.output_csv, format='csv', overwrite=True)
         print('csv exposure list written to {}'.format(self.output_csv))
 
+
+    def check_aperture_override(self):
+        instruments = self.exposure_tab['Instrument']
```

Comment:
Should this function not check for the `FiducialPointOverride` key somehwere and only act if that is `True`? Maybe I am misunderstanding.

Reply:
Good point! Done. Thanks for pointing that out.